massdriver-application/aws_identity.tf

```diff
@@ -33,7 +33,7 @@ locals {
 }
 EOF
 
-  aws_federated_principal_assume_role = <<EOF
+  aws_federated_principal_assume_role = (local.aws_eks_oidc_short == null || local.aws_eks_assume_role_conditional == null) ? null : <<EOF
```

Comment:
if either of these values are null, we don't want to interpolate them. They are null if we are using this with any other cloud.